libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -2004,7 +2004,7 @@ void AP_OSD_Screen::draw_esc_temp(uint8_t x, uint8_t y)
 {
     int16_t etemp;
     // first parameter is index into array of ESC's.  Hardwire to zero (first) for now.
```

Comment:
Comment is now wrong. I also think you should update the X Y params for this to explain its now the highest.

Reply:
Fixed